src/main/java/edu/cornell/kfs/fp/service/impl/RecurringDisbursementVoucherDocumentServiceImpl.java

```diff
@@ -311,7 +339,72 @@ private RecurringDisbursementVoucherPDPStatus buildRecurringDisbursementVoucherP
         }
         return paymentDetails;
     }
- 
+    
+    private void noteChangeOnRecurringDV(RecurringDisbursementVoucherDocument recurringDV, String noteText, Set<String> setOfStrings) {
+        if (setOfStrings != null && !setOfStrings.isEmpty()) {
```

Comment:
You don't need the null check if you're certain that this private method is only being invoked with a non-null Set. Alternatively, you could use CollectionUtils.isNotEmpty() to keep things simpler.